.gitignore

```diff
@@ -29,6 +29,9 @@ log
 dask-worker-space/
 tmp/
 
+# Files generated from CMake Templates
+cpp/cmake/Find*.cmake
```

Comment:
What if we wanted to really write a `cpp/cmake/FindSomething.cmake` manually?

Reply:
Good point, will add the ones we want to exclude explicitly

Reply:
The Find*.cmake files are now placed in the `build` directory, so they don't need to be added here and not create artifacts outside the build directory

Reply:
Thanks